### PR TITLE
Add Github Reviewers

### PR DESCRIPTION
**Pull request by Sayan-**
When we first added the Github integration, the `reviewers` endpoints had not been completed for `Github Apps`. It has since been added, so let's do this!

Changes:
* Added a `AddReviewers` method
* Added bot level test cases for `setAssignee`
* Changed the parser to accept things in the form of: `github.com/{ORG_NAME}/{REPO}/pull/...` (feedback from @a-le-jan-dro )

API Validation Testing: check it ⬇️

### CODE REVIEW

File: bot.go [hunk 1]
```diff
@@ -20,6 +20,7 @@ import (
 
 // generate mocks of dependencies for use during testing
 //go:generate sh -c "$PWD/bin/mockgen -package main -source $PWD/slackapi/SlackService.go SlackAPIService,SlackRTMService > slack_service_mock_test.go"
+//go:generate sh -c "$PWD/bin/mockgen -package main -source $PWD/github/client.go AppClientIface > github_mock_test.go"
```
Comment (Sayan-) [inline bot.go]: Another pattern I've seen recently is to keep all mocks in a `mocks` package: https://github.com/Clever/catapult/blob/master/mocks/gen.go#L5. I was wondering which pattern you prefer in general :)

Comment (nathanleiby) [reply bot.go]: No strong opinion, this LGTM!

**Comment by nathanleiby** [comment]
Great stuff! 

The only other improvement I think could  be handy is a UX nit-- might pickabot say something slightly different when assigning (YES assignee+reviewer) vs just picking (NO assign/reviewer).

That way users can see the difference.

**Comment by Sayan-** [comment]
Ah great suggestion! I'll add a message in the `setAssignee` code!

**Pull request merged**